.github/workflows/publish-and-deploy-gamma.yaml

```diff
@@ -0,0 +1,36 @@
+name: Publish and Deploy Gamma
+
+on:
+  push:
+    branches: [ dev ]
+  workflow_dispatch:
+    branches:
```

Comment:
My reading of the docs say that we are doing this incorrectly in various places, likely my fault. https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/manually-running-a-workflow